openpype/hosts/flame/plugins/publish/extract_subset_resources.py

```diff
@@ -131,161 +165,157 @@ def process(self, instance):
-                ):
-                    representation_data["files"] = files.pop()
-                else:
-                    representation_data["files"] = files
-
-                # add frame range
-                if preset_config["representation_add_range"]:
-                    representation_data.update({
-                        "frameStart": frame_start_handle,
-                        "frameEnd": (
-                            frame_start_handle + source_duration_handles),
-                        "fps": instance.data["fps"]
-                    })
-
-                instance.data["representations"].append(representation_data)
-
-                # add review family if found in tags
-                if "review" in repre_tags:
-                    instance.data["families"].append("review")
-
-                self.log.info("Added representation: {}".format(
-                    representation_data))
+            # create preset path
+            preset_orig_xml_path = str(os.path.join(
+                preset_dir, preset_file
+            ))
+
+            preset_path = opfapi.modify_preset_file(
+                preset_orig_xml_path, staging_dir, modify_xml_data)
+
+            # define kwargs based on preset type
+            if "thumbnail" in unique_name:
+                export_kwargs["thumb_frame_number"] = in_mark + (
+                    source_duration_handles / 2)
+            else:
+                export_kwargs.update({
+                    "in_mark": in_mark,
+                    "out_mark": out_mark
+                })
+
+            # get and make export dir paths
+            export_dir_path = str(os.path.join(
+                staging_dir, unique_name
+            ))
+            os.makedirs(export_dir_path)
+
+            # export
+            opfapi.export_clip(
+                export_dir_path, exporting_clip, preset_path, **export_kwargs)
+
+            # create representation data
+            representation_data = {
+                "name": unique_name,
+                "outputName": unique_name,
+                "ext": extension,
+                "stagingDir": export_dir_path,
+                "tags": repre_tags,
+                "data": {
+                    "colorspace": color_out
+                },
+                "load_to_batch_group": load_to_batch_group,
+                "batch_group_loader_name": batch_group_loader_name
+            }
+
+            # collect all available content of export dir
+            files = os.listdir(export_dir_path)
+
+            # make sure no nested folders inside
+            n_stage_dir, n_files = self._unfolds_nested_folders(
+                export_dir_path, files, extension)
+
+            # fix representation in case of nested folders
+            if n_stage_dir:
+                representation_data["stagingDir"] = n_stage_dir
+                files = n_files
+
+            # add files to represetation but add
+            # imagesequence as list
+            if (
```

Comment:
I think don't we support to have multiple files in `representation["files"]` (like 2 and more .move files).
So this could be simplified to:
```
if len(files) == 1:
    representationdata["files"] = files[0]
else:
    representationdata["files"] = files
```